server/src/routes/index.js

```diff
@@ -4,6 +4,7 @@ const router = require("express").Router();
 // Custom Modules
 const InfoCtrl = require("../controllers/info.controller");
 const GameCtrl = require("../controllers/game.controller");
+const newUser = require("../controllers/userProfile.controller.js")
```

Comment:
make the name of this constant consistent with the others e.g `GameCtrl`. yours should be `UserCtrl`